### PR TITLE
DT-398 Update pull_request_template.md

### PR DESCRIPTION
**Pull request by aisayo**
### Jira Ticket 
https://ombulabs.atlassian.net/browse/DT-398

### Motivation and Context
- Update pull_request_template.md to include a template that will maintain consistency across PR requests for Points.
- Updated included adding the following sections:
  - Jira Ticket
  - Motivation and Context 
  - QA and testing instructions
  - Screenshots

### QA and Testing Instructions
- For QA, please review the pull_request_template.md file and leave suggestions for improvement
___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+### Jira Ticket 
+<!--- Add a link to the Jira ticket associated with this PR -->
 
-Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behavior are made, clearly describe what changes.
-If changes to the UI are made, please include screenshots of the before and after.
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- Describe the approach to solve the problem or complete the task -->
+    
+### QA and Testing Instructions
```
Comment (fbuys) [inline pull_request_template.md:10]: To make it shorter, what do you think?
```suggestion
### QA / Testing Instructions
```

File: pull_request_template.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
 **IMPORTANT: Please read the README before submitting pull requests for this project. Additionally, if your PR closes any open GitHub issues, make sure you include _Closes #XXXX_ in your comment or use the option on the PR's sidebar to add related issues to auto-close the issue that your PR fixes. **
 
-**Description:**
+### Jira Ticket 
+<!--- Add a link to the Jira ticket associated with this PR -->
 
-Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behavior are made, clearly describe what changes.
-If changes to the UI are made, please include screenshots of the before and after.
+### Motivation and Context
```
Comment (fbuys) [inline pull_request_template.md:6]: Also to make it shorter:
```suggestion
### Motivation / Context
```

File: pull_request_template.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+<!--- Why is this change required? What problem does it solve? -->
+<!--- Describe the approach to solve the problem or complete the task -->
```
Comment (fbuys) [inline pull_request_template.md:8]: Perhaps we can remove this line?

File: pull_request_template.md [hunk 4]
```diff
@@ -1,9 +1,17 @@
 **IMPORTANT: Please read the README before submitting pull requests for this project. Additionally, if your PR closes any open GitHub issues, make sure you include _Closes #XXXX_ in your comment or use the option on the PR's sidebar to add related issues to auto-close the issue that your PR fixes. **
 
-**Description:**
+### Jira Ticket 
+<!--- Add a link to the Jira ticket associated with this PR -->
 
-Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behavior are made, clearly describe what changes.
-If changes to the UI are made, please include screenshots of the before and after.
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
```
Comment (fbuys) [inline pull_request_template.md:7]: If we update this line to something like this:
```suggestion
<!--- Why is this change required? Summarize the changes you made.  -->
```

File: pull_request_template.md [hunk 5]
```diff
@@ -1,9 +1,17 @@
...
+<!--- Describe the approach to solve the problem or complete the task -->
+    
+### QA and Testing Instructions
+<!--- Include any relevant details about testing and QA steps to be followed -->
```
Comment (fbuys) [inline pull_request_template.md:11]: How does this sound? Any better or clearer?
`List the steps required to review the changes in locally or in staging`

**Review by fbuys — COMMENTED** [review]
Looking good, I made some notes :-)

File: pull_request_template.md [hunk 6]
```diff
@@ -1,9 +1,15 @@
...
+### Jira Ticket 
+<!--- Add a link to the Jira ticket associated with this PR -->
 
-**Description:**
+### Motivation / Context
+<!--- Why is this change required? Summarize the changes you made.  -->
+<!--- Describe the approach to solve the problem or complete the task -->
+    
+### QA / Testing Instructions
+<!--- List the steps required to review the changes in locally or in staging -->
```
Comment (fbuys) [inline pull_request_template.md:9]:
```suggestion
<!--- List the steps required to review the changes locally or in staging -->
```

**Review by rishijain — APPROVED** [review]

**Comment by rishijain** [comment]
the template lgtm :tada:

**Review by fbuys — APPROVED** [review]

**Pull request merged**